examples/guestbook/README.md

```diff
@@ -199,81 +276,51 @@ If you want to see the logs for a given container, you can run:
 $ docker logs <container_id>
 ```
 
-### Step Two: Fire up the redis master service
+### Step Three: Create the redis slave service
```

Comment:
You mean refer to https://github.com/kubernetes/kubernetes/blob/master/docs/user-guide/managing-deployments.md in the combine step 3 and 4?